app/managers/network_manager.ts

```diff
@@ -19,18 +20,41 @@ import {Client} from '@client/rest';
 import * as ClientConstants from '@client/rest/constants';
 import ClientError from '@client/rest/error';
 import {CERTIFICATE_ERRORS} from '@constants/network';
-import {DEFAULT_LOCALE, getLocalizedMessage, t} from '@i18n';
+import {DEFAULT_LOCALE, getTranslations} from '@i18n';
 import ManagedApp from '@init/managed_app';
+import {toMilliseconds} from '@utils/datetime';
 import {logDebug, logError} from '@utils/log';
 import {getCSRFFromCookie} from '@utils/security';
 
 const CLIENT_CERTIFICATE_IMPORT_ERROR_CODES = [-103, -104, -105, -108];
 const CLIENT_CERTIFICATE_MISSING_ERROR_CODE = -200;
 const SERVER_CERTIFICATE_INVALID = -299;
+const SERVER_TRUST_EVALUATION_FAILED = -298;
+let showingServerTrustAlert = false;
+
+const messages = defineMessages({
+    invalidSslTitle: {
+        id: 'server.invalid.certificate.title',
+        defaultMessage: 'Invalid SSL certificate',
+    },
+    invalidSSLDescription: {
```

Comment:
Inconsistent casing consistent between `invalidSslTitle` and `invalidSSLDescription` (one of them is SSL and the other is Ssl)